File: Python/gerador_senhas.py
import random

from numpy import number

def generatePassword(length):
    i = 0
    char = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%&*()-+=`´{[}]^;~:.,'
    password = ''
    
    while i < length:
        password += random.choice(char)
        i += 1
    
    print("Senha gerada: " + password)

print("==========[ GERADOR DE SENHA ALEATÓRIA EM PYTHON ]==========")
while True:
    length = int(input("Quantos caracteres deverá ter a senha?: "))
    generatePassword(length)
    print("----------------------------------")

File: C++/gerador_senhas.cpp
#include <iostream>
#include <time.h>

using namespace std;
char chars[88] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!@#$%&*()-+=`{[}]^;~:.,"; 
int number, length;

int main(int argc, char const *argv[])
{
    srand(time(0));
    cout << "==========[ GERADOR DE SENHAS EM C++ ]==========" << endl;
    while(true){
        cout << endl << "Quantidade de caracteres da senha?: ";
        cin >> length;
        cout << "Senha: ";
        for(int i = 0; i < length; i++){
            number = (rand() % 88) + 1;
            cout << chars[number];
        }
        cout << endl;
    }
    return 0;
}
